docs/UserGuide.md

```diff
@@ -146,8 +146,11 @@ Adds a student to the address book.
 **Example:**
 * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 t/Sunday-1000-1200 s/Geography r/100 paid/100 owed/0`
 
-**Output:**
-![addResult.jpeg](images/addResult.jpeg)
+**Before (using an empty UGTeach):**
+![addBefore.png](images/addBefore.png)
+
+**After:**
+![addBAfter.png](images/addAfter.png)
```

Comment:
Minor error here, addBAfter should be addAfter

Reply:
Thanks for pointing this out! Ive fixed this